docs/dictionary/message/pasteKey.lcdoc

```diff
@@ -18,9 +18,10 @@ Platforms: desktop, server
 
 Example:
 on pasteKey -- replace returns with spaces before pasting
-  if the clipboardData["text"] is not empty then set \
-     the clipboardData["text"] to \
+  if the clipboardData["text"] is not empty then 
+     set the clipboardData["text"] to \
      replaceText(the clipboardData["text"],return,space)
```

Comment:
This line should be indented, so as it appears like this:
```
if the clipboardData["text"] is not empty then 
   set the clipboardData["text"] to \
         replaceText(the clipboardData["text"],return,space)
end if
```

(You can paste it in the LC script editor to see how it appears there)